### DISCUSSION TITLE
Обговорення дизайну платформи

### DISCUSSION THREAD

**Discussion by shaitanaleksander** [ratifire/devrate-service#23] [Polls]
Привіт! ) Допоможіть вибрати стиль дизайну.

Референси (розбила на декілька блоків з більш-менш схожим стилем):
(повідомлення скопійоване)

1. https://www.uidino.com/ . https://www.framer.com/?via=dribbble.
2. https://quixel.com/. https://www.unrealengine.com/en-US
3. https://www.feyapp.com/. https://pad.dotincorp.com/. https://podcast.humbleteam.com/.
4. https://djinni.co/. https://interviewing.io/faq
5. https://webflow.com/?r=0

**Comment by olena-diachenko** [comment]
Як на мене, в більшості дизайнів забагато анімації, вона дуже відволікає увагу.

**Reply by IvashchenkoIvanIgorevich** [reply]
Згоден, але є і такі як https://djinni.co/ на якому дизайн виглядає застарілим, на мою думку.

**Reply by olena-diachenko** [reply]
Згодна.

**Reply by shaitanaleksander** [reply]
не звертайте увагу на анімаціі  спочатку іх точно не буде. головне це стиль сайту (колір розмітка вигляд меню ....)

**Comment by ghost** [comment]
Так, обираємо просто стиль … в налаштуваннях «кабінету», можна буде передбачити можливість обирати темну чи світлу тему.

**Comment by ghost** [comment]
<img width="1458" alt="Знімок екрана 2024-01-11 о 16 33 49" src="https://github.com/ratifire/devrate-service/assets/156097303/a9162f0f-ee13-473a-a8f3-040de5c91daa">

**Reply by ghost** [reply]
наприклад ...сторінку можна залишити ось так статичною, а в подальшому проанімувати ..стиль від цього не змінюється

**Comment by hubersky** [comment]
Перші найкращі як на мене, особливо 2 варіант , можливо зробити колористику як там у 2 варіанті або як тут в прикладі, мені  подобається такий стиль:
![photo_2024-01-11_20-07-42](https://github.com/ratifire/devrate-service/assets/95440212/246aeaa0-3b0d-4453-b207-7384afe57a5c)

**Reply by hubersky** [reply]
Можна в майбутьньому дві теми зробити світлу і темну як всі і привикли на топових площадках, а почати з чогось одного.

**Reply by hubersky** [reply]
Ще 3 варіант той що помаранчевий теж класний і сучасний, я би теж за нього проголосував)

**Comment by weistapki** [comment]
Netflix ) небо 2 дуже нiчого.
